classes/basement/rootrecipe.yaml

```diff
@@ -38,6 +38,9 @@ depends:
           - name: devel::m4
             use: [tools]
             forward: True
+          - name: devel::help2man
```

Comment:
Could you add it to the list of available tools in `README.md` and `doc/usage.rst`?